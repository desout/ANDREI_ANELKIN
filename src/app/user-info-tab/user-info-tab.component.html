<div class="container">
<ng-container *ngIf="(user ); then info; else loading"></ng-container>
<ng-template #info>
   <div class="form-group row">
     <label class="col-form-label">{{'INFO.USERNAME' | translate}}
       <div class="col-sm-10">
       <input type="text" readonly class="form-control-plaintext"  value="{{ user.name }}">
     </div>
     </label>
   </div>
  <div class="form-group row">
    <label class="col-form-label">{{'INFO.BIRTHDAY' | translate}}
      <div class="col-sm-10">
        <input type="text" readonly class="form-control-plaintext"  value="{{ user.dateOfBirth }}">
      </div>
    </label>
  </div>
  <div class="form-group row">
    <label class=" col-form-label">{{'INFO.DATEOFLOGIN' | translate}}
      <div class="col-sm-10">
        <input type="text" readonly class="form-control-plaintext"  value="{{ user.dateOfFirstLogin }}">
      </div>
    </label>
  </div>
  <div class="form-group row">
    <label class="col-form-label">{{'INFO.DATEOFNEXTNOTIF' | translate}}
      <div class="col-sm-10">
        <input type="text" readonly class="form-control-plaintext"  value="{{ user.dateNextNotification }}">
      </div>
    </label>
  </div>
  <div class="form-group row">
    <label class="col-form-label">{{'INFO.INFORMATION' | translate}}
      <div >
        <input type="text" readonly class="form-control-plaintext"  value="{{ user.information }}">
      </div>
    </label>
  </div>

</ng-template>
<ng-template #loading><div class='lmask'></div></ng-template>
</div>
