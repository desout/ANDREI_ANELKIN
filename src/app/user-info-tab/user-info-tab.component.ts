import { Component, OnInit } from '@angular/core';
import {TranslateService} from '@ngx-translate/core';
import {select, Store} from '@ngrx/store';
import { CurUserState} from '../store';
import {UserService} from '../user.service';

@Component({
  selector: 'app-user-info-tab',
  templateUrl: './user-info-tab.component.html',
  styleUrls: ['./user-info-tab.component.scss']
})
export class UserInfoTabComponent implements OnInit {
  constructor(public translate: TranslateService, public userService: UserService, public sessionStore: Store<CurUserState>) { }

  ngOnInit() {
  }


}
