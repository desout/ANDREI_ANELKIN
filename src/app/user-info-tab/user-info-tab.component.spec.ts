import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { UserInfoTabComponent } from './user-info-tab.component';
import {NO_ERRORS_SCHEMA} from '@angular/core';
import {TestModule} from '../Test.module';

describe('UserInfoTabComponent', () => {
  let component: UserInfoTabComponent;
  let fixture: ComponentFixture<UserInfoTabComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [],
      schemas: [NO_ERRORS_SCHEMA],
      imports: [TestModule],
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserInfoTabComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
