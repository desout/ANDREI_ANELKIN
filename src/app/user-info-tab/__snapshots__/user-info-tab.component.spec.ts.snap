// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfoTabComponent should create 1`] = `
<div
  id="root0"
  ng-version="7.0.3"
>
  <div
    class="container"
  >
    
    <div
      class="lmask"
    />
    
    
  </div>
</div>
`;
